modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ContainerFluidTag.java

```diff
@@ -26,7 +28,10 @@ public int doStartTag() throws JspException {
 		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
 
 		setFluid(true);
-		setSize("xl");
+
+		if (Validator.isNull(getSize())) {
```

Comment:
Weird. we were receiveing size property but we aren't using this value inside the container-fluid tag, right?

Reply:
~~When reading https://getbootstrap.com/docs/4.4/layout/overview/#fluid, we don't have a class like `container-fluid-xl`.~~

Nevermind, we are implementing this on clay side: https://clayui.com/docs/css/grid.html#container-fluid  :)

Reply:
Yes, before this change we are ignoring the size attribute and we have the hardcoded size `xl` always.